### PR TITLE
add link to arch linux packages

### PR DESCRIPTION
**Pull request by wilzbach**
I think they should be good to go now.

### CODE REVIEW

**Comment by 9il** [comment]
We should add information about DUB, because it is not obvious for newcomers that DUB is package manager for D

**Comment by wilzbach** [comment]
Yes, but that's a different concern.
Once this package is installed, `dub` is not needed ;-)

**Comment by 9il** [comment]
> Yes, but that's a different concern.
> Once this package is installed, dub is not needed ;-)

What would think Windows user when he saw Arch package and no words about Windows? He would think that this is only for Arch linux and bearded engineers

**Comment by 9il** [comment]
I mean that we need to add few words, that mir can be easily used on all operating systems using dub

**Comment by wilzbach** [comment]
> Looks better, imho

Yep :)

> I mean that we need to add few words, that mir can be easily used on all operating systems using dub

Can you check what I wrote?

File: README.md [hunk 1]
```diff
@@ -52,3 +52,43 @@ Numeric library and mirror for upcoming numeric packages for the Dlang standard
...
+
+### Arch Linux
+
+| Type   | Name         | Version |
+|--------|--------------|----------|
+| Stable | `libmir`     | [![libmir](https://img.shields.io/aur/version/libmir.svg)](https://aur.archlinux.org/packages/libmir/) |
+| Latest | `libmir-git` | [![libmir-git](https://img.shields.io/aur/version/libmir-git.svg)](https://aur.archlinux.org/packages/libmir-git/) |
+
+
+### On any platform with dub
```
Comment (9il) [inline README.md:58]: This should be the first section

File: README.md [hunk 2]
```diff
@@ -52,3 +52,43 @@ Numeric library and mirror for upcoming numeric packages for the Dlang standard
...
+
+[Dub](https://code.dlang.org/getting_started) is the D's package manager.
+You can create a new project with:
+
+```
+dub init
```
Comment (9il) [inline README.md]: `dub init <project-name>`

File: README.md [hunk 3]
```diff
@@ -52,3 +52,43 @@ Numeric library and mirror for upcoming numeric packages for the Dlang standard
...
+```
+
+Now you need to edit the `dub.sdl` add `mir` as dependency:
+
+```
+dependency "mir" version="~><current-version>"`
```
Comment (9il) [inline README.md]: Please define docs for `dub.json` instead of sdl

Comment (wilzbach) [inline README.md]: `dub init` will create `sdl`

File: README.md (hunk 2 above)
Comment (wilzbach) [inline README.md]: `dub init` will create the `sdl` in the current folder.

File: README.md (hunk 1 above)
Comment (9il) [inline README.md:58]: please put DUB with version shield after section title

File: README.md (hunk 3 above)
Comment (9il) [inline README.md]: `dub init --format=json`

File: README.md [hunk 4]
```diff
@@ -52,3 +52,49 @@ Numeric library and mirror for upcoming numeric packages for the Dlang standard
...
+
+### On any platform with dub
+
+[![Dub version](https://img.shields.io/dub/v/mir.svg)](http://code.dlang.org/packages/mir)
+
+[Dub](https://code.dlang.org/getting_started) is the D's package manager.
+You can create a new project with:
+
+```
+dub init --format=json
```
Comment (9il) [inline README.md:66]: `dub init --format=json <project-name>`

File: README.md [hunk 5]
```diff
@@ -52,3 +52,49 @@ Numeric library and mirror for upcoming numeric packages for the Dlang standard
...
+|--------|--------------|----------|
+| Stable | `libmir`     | [![libmir](https://img.shields.io/aur/version/libmir.svg)](https://aur.archlinux.org/packages/libmir/) |
+| Latest | `libmir-git` | [![libmir-git](https://img.shields.io/aur/version/libmir-git.svg)](https://aur.archlinux.org/packages/libmir-git/) |
+
+### On any platform with import paths
+
+Mir is a pure source code library, that means it can be easily distributed to
+any system. So you can just copy Mir's source to your system's dlang import path.
+For example on  Linux this is `/usr/include/dlang/dmd/mir`
+(or ``/usr/include/dlang/dmd/mir` for ldc).
```
Comment (9il) [inline README.md:98]: ``/usr

**Comment by wilzbach** [comment]
Hey I still wanted to address the issues :/
Will submit follow up..

**Comment by 9il** [comment]
Don't worry :-) I like your new readme )

**Comment by wilzbach** [comment]
No I am sorry that I didn't fix it as sometimes there are really times when I am not at my laptop :/

**Pull request merged**